src/views/v2/components/fields/radio.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/components/fields/radio.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Radio
+ */
+
+$field      = $this->get( 'field' );
+$field_name = tribe_tickets_ar_field_name( $ticket->ID, $field->slug );
+$required   = tribe_tickets_ar_field_is_required( $field );
+$field      = (array) $field;
+$options    = null;
+
+if ( isset( $field['extra'] ) && ! empty( $field['extra']['options'] ) ) {
+	$options = $field['extra']['options'];
+}
+
+if ( ! $options ) {
+	return;
+}
+
+$value    = '';
+$disabled = false;
+$slug     = $field['slug'];
+
+$classes = [
+	'tribe-tickets__form-field',
+	'tribe-tickets__form-field--required' => $required,
+];
+?>
+<div <?php tribe_classes( $classes ); ?>>
+	<header class="tribe-tickets__form-field-label">
+		<h3 class="tribe-common-b1 tribe-common-b2--min-medium">
+			<?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?>
+		</h3>
+	</header>
+
+	<div class="tribe-common-form-control-checkbox-radio-group">
+		<?php
+		foreach ( $options as $option ) :
+			$option_slug = md5( sanitize_title( $option ) );
+			$field_slug  = $field['slug'];
+			$option_id   = tribe_tickets_ar_field_id( $ticket->ID, $field_slug, $option_slug );
+			$slug        = $field_slug . '_' . $option_slug;
+			$value       = [];
```

Comment:
Where do we set `$value`? We call it below to check if `$slug` is in it but never fill up the `$value` array.

Reply:
I didn't really worked on the logic of these fields. Just grabbed what we have now, working for the AR fields and moved them to a generic folder in order to componetize them. Adding generic form CSS classes and using a few new template tags. But haven't worked on the logic.